.eslintrc.json

```diff
@@ -45,6 +45,7 @@
         ],
         "react/prop-types": [0, {}],
         "no-unused-vars": "off",
-        "@typescript-eslint/no-unused-vars": "warn"
+        "@typescript-eslint/no-unused-vars": "warn",
+        "no-console": "warn",
```

Comment:
Trailing comma causes errors in json files (js files are fine)
Different apps seem to handle it differently, when there was an extra comma in tsconfig.json I got some super specific non related error at some point in the future

Reply:
Also nice!